bc_obps/compliance/api/_compliance_report_versions/_compliance_report_version_id/manual_handling.py

```diff
@@ -0,0 +1,64 @@
+from typing import Optional, Tuple
+from django.http import HttpRequest
+from common.api.utils.current_user_utils import get_current_user
+from compliance.api.router import router
+from compliance.constants import COMPLIANCE
+from registration.schema.generic import Message
+from service.error_service.custom_codes_4xx import custom_codes_4xx
+from compliance.api.permissions import (
+    approved_authorized_roles_compliance_report_version_composite_auth,
+    approved_industry_user_cas_director_cas_analyst_compliance_report_version_composite_auth,
+)
+from compliance.models import ComplianceReportVersionManualHandling
+from compliance.schema.compliance_report_version_manual_handling import (
+    ComplianceReportVersionManualHandlingOut,
+    ComplianceReportVersionManualHandlingIn,
+)
+from compliance.service.compliance_report_version_manual_handling_service import ComplianceManualHandlingService
+
+
+@router.get(
+    "/compliance-report-versions/{compliance_report_version_id}/manual-handling",
+    response={200: ComplianceReportVersionManualHandlingOut, custom_codes_4xx: Message},
+    tags=COMPLIANCE,
+    description="Get manual handling data for a compliance report version",
+    exclude_none=True,
+    auth=approved_authorized_roles_compliance_report_version_composite_auth,
```

Comment:
Should this be: `auth=approved_industry_user_cas_director_cas_analyst_compliance_report_version_composite_auth,`